tester_app/client/webpack.client.base.config.js

```diff
@@ -0,0 +1,58 @@
+// Common client-side webpack configuration used by webpack.hot.config and webpack.rails.config.
+
+const webpack = require('webpack');
+const path = require('path');
+
+module.exports = {
+
+  // the project dir
+  context: __dirname,
+  entry: {
+
+    // See use of 'vendor' in the CommonsChunkPlugin inclusion below.
+    vendor: [
+      'babel-core/polyfill',
+      'jquery',
+      'jquery-ujs',
+      'react',
+      'react-dom',
+    ],
+
+    // This will contain the app entry points defined by webpack.hot.config and webpack.rails.config
+    app: [
+      './app/bundles/HelloWorld/startup/clientGlobals',
+    ],
+  },
+  resolve: {
+    extensions: ['', '.webpack.js', '.web.js', '.js', '.jsx', '.scss', '.css', 'config.js'],
+    alias: {
+      lib: path.join(process.cwd(), 'app', 'lib'),
```

Comment:
What @jbhatab said. It's just different ways of do one thing. I choose this one because thus I unambiguously declare that `lib === /path/to/lib` so webpack doesn't have to resolve modules in array of paths every time it's imported.